test/unit/jwplayer-selectplayer-test.js

```diff
@@ -4,28 +4,36 @@ define([
     'jwplayer'
 ], function (_, $, jwplayer) {
 
+    function testInstanceOfApi(assert, api) {
+        assert.isOk(_.isObject(api), 'jwplayer({dom id}) returned an object');
+        assert.isOk(_.isFunction(api.setup), 'object.setup is a function');
```

Comment:
@robwalch when you write new or update unit tests, switch `assert` to `expect`. Follows BDD and makes easier to read. You need to pass around `expect` everywhere as well.

http://chaijs.com/api/bdd/